docs/en/reference/working-with-associations.rst

```diff
@@ -488,30 +471,52 @@ and the "remove" operation.
         //...
     }
 
-Since ``cascade: persist`` is configured for the ``User#commentsAuthored``
-association, you can now create a user and persist their comments as follows:
+...you can now create a user and an associated comment like this:
 
 .. code-block:: php
 
     <?php
     $user = new User();
-    $myFirstComment = new Comment();
-    $user->addComment($myFirstComment);
-    $myFirstComment->setUser($user);
+    $user->comment('Lorem ipsum', new DateTime());
     
     $em->persist($user);
     $em->flush();
 
+Thanks to ``cascade: remove``, you can easily delete a user too (without having to loop through all associated comments):
+
+.. code-block:: php
+
+    <?php
+    $user = $em->find('User', $deleteUserId);
+
+    $em->remove($user);
+    $em->flush();
+
 .. note::
 
-    This code does not associate the newly created comment with the user.
-    To achieve this, you *always* have to call ``$myFirstComment->setUser($user);`` –
-    no matter if ``cascade: persist`` is set or not.
+    The idea of ``cascade: persist`` is not to save you any lines of code in the controller.
```

Comment:
I'd move the note about `cascade: persist` up above the section on `cascade: remove` to keep the topics more separated.